config-linux.md

```diff
@@ -625,6 +625,22 @@ The following parameters can be specified to set up seccomp:
     * `SECCOMP_FILTER_FLAG_LOG`
     * `SECCOMP_FILTER_FLAG_SPEC_ALLOW`
 
+* **`listenerPath`** *(string, OPTIONAL)* - specifies the path of UNIX domain socket over which the runtime will send the [container process state](#containerprocessstate) data structure when the `SCMP_ACT_NOTIFY` action is used.
+    This socket MUST use `AF_UNIX` domain and `SOCK_STREAM` type.
+    The runtime MUST send exactly one [container process state](#containerprocessstate) per connection.
+    The connection MUST NOT be reused and it MUST be closed after sending a seccomp state.
+    If sending to this socket fails, the runtime MUST [generate an error](runtime.md#errors).
```

Comment:
Do we want to allow for "fail-open" or "fail-closed behaviour?

Reply:
There should be a version or something, given that container process state appears to be extensible. The receiving end has to know how many FDs to receive *before* it receives them. Unless we have some kind of negotiation mechanism which first sends the version to enable the receiving end to know what to expect.

Reply:
> Do we want to allow for "fail-open" or "fail-closed behaviour?

IMO I think it should fail if sending the message to the seccomp agent fails for some reason (e.g. it is not running, or permission denied to connect to the socket). In that way, we don't make debugging difficult by accidentally starting containers that would return ENOSYS for some system calls.

I want to be able to run a seccomp agent in the following setups:
- as a socket activatable systemd service. If it is not started yet, it would be started on demand
- as a DaemonSet on Kubernetes. If it is not started yet, other pods that need the seccomp agent would fail. But pods that don't need it (if they use a seccomp policy without any `SCMP_ACT_NOTIFY` rule), they would start fine. So in that case, the seccomp agent DaemonSet would not have a chicken-egg problem.

Reply:
Regarding not failing on sending to the unix socket, what is the point of making it not fail? I see only downsides, it makes no sense to start containers and just throw the seccomp fds it needs to work to `/dev/null`. Am I missing something?

Reply:
@sargun marking as solved as you recently reviwed with a LGTM. Please feel free to reopen if you think this isn't addressed :)